tests/foreman/cli/test_bootstrap_script.py

```diff
@@ -53,6 +53,7 @@ def test_positive_register(
 
     :BZ: 2001476
     """
+    rhel_contenthost.enable_ipv6_dnf_and_rhsm_proxy()
```

Comment:
Should we document that this is a workaround for the following two things:
* A bug in bootstrap.py: https://github.com/Katello/katello-client-bootstrap/pull/373
* The fact that `rhel_contenthost` has internet-based repos enabled, which it can't reach in IPv6-only setups, but also shouldn't have configured to begin with

(And then remove it once we fixed `rhel_contenthost`?)